modules/default/calendar/calendar.css

```diff
@@ -1,7 +1,4 @@
 .calendar .symbol {
-  display: flex;
```

Comment:
Because the flex causes this to screw up and generate extra empty space
![image](https://user-images.githubusercontent.com/18092668/218327726-8fe6cfcb-cb05-41bd-93eb-63e418afbc59.png)

Reply:
I can put it back... but it wouldn't look pleasing on the eyes

Reply:
And like I said it would break existing layouts, like mine:

![grafik](https://user-images.githubusercontent.com/1313547/219098903-f0dfd185-5827-449d-958c-9040a26ec3fb.png)

It might be necessary to cleanup the layout of the calendar module, but not in this PR :-)

Reply:
Very well then, I will put the changes back and make a temporary work around for myself... I don't want to hold of this PR any longer than is needed

Reply:
Thanks. I do like your PR and the screenshot looks nice, but we got to take care of exisiting users. Thanks for you understanding